src/lib/data.ts

```diff
@@ -205,7 +214,27 @@ async function get_test_metadata(dir_name: string): Promise<TestData[]> {
                 return [fallback]
             }
         };
-    
+
+        const get_array_of_meta_values = (property: string, metadata: any): string[] => {
+            return metadata["meta"].filter((entry:any): boolean => entry["$"].property === property)
+        }
+
+        const get_single_meta_value = (property: string, metadata: any): any => {
+            return metadata.meta.find((entry: any): boolean => entry["$"].property === property)
+        }
+
+        const is_required = (metadata: any): boolean => {
```

Comment:
I think we should pass through the actual value here (maybe all-capsing it). I would like to see MUST or SHOULD or MAY.

If this were a Google review, I would link to the style-guide section that says "Never use booleans for something that has two states now, but might have more later." I have found this to be very good advice :)

Reply:
I changed it to use "must", "should", or "may". More exactly, I have defined a literal type that can take one of these three values.